Unable to Install the Package - gyp ERR! find Python

I am using Node v16.17.0 and every time I try to install the package, I am getting the following error. Do I need to install python?

> npm ERR! path D:\my-app\node_modules\@curlconverter\tree-sitter-bash
> npm ERR! command failed
> npm ERR! command C:\Windows\system32\cmd.exe /d /s /c prebuild-install || node-gyp rebuild
> npm ERR! prebuild-install warn install No prebuilt binaries found (target=16.17.0 runtime=node arch=x64 libc= platform=win32)
> npm ERR! gyp info it worked if it ends with ok
> npm ERR! gyp info using node-gyp@9.0.0
> npm ERR! gyp info using node@16.17.0 | win32 | x64
> npm ERR! gyp ERR! find Python

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "curlconverter",
-  "version": "4.7.0",
+  "version": "4.7.1",
   "description": "convert curl commands to Python, JavaScript, Go, PHP and more",
   "homepage": "https://curlconverter.com",
   "author": {
@@ -25,7 +25,7 @@
   "type": "module",
   "dependencies": {
     "@curlconverter/tree-sitter": "^0.0.10",
-    "@curlconverter/tree-sitter-bash": "^0.0.2",
+    "@curlconverter/tree-sitter-bash": "^0.0.3",
     "jsesc": "^3.0.2",
     "lossless-json": "^2.0.8",
     "web-tree-sitter": "^0.20.8",

--- a/src/cli.ts
+++ b/src/cli.ts
@@ -56,7 +56,7 @@ import { _toWget, toWgetWarn } from "./generators/wget.js";
 import fs from "fs";
 
 // This line is updated by extract_curl_args.py
-const VERSION = "4.7.0 (curl 8.0.1)";
+const VERSION = "4.7.1 (curl 8.0.1)";
 
 // sets a default in case --language isn't passed
 const defaultLanguage = "python";

--- a/src/shell/tokenizer.ts
+++ b/src/shell/tokenizer.ts
@@ -112,10 +112,10 @@ function toTokens(
       return [removeBackslashes(node.text)];
     case "raw_string":
       return [node.text.slice(1, -1)];
-    case "ansii_c_string":
+    case "ansi_c_string":
       return [removeAnsiCBackslashes(node.text.slice(2, -1))];
     case "string":
-    case "string_expansion": {
+    case "translated_string": {
       // TODO: MISSING quotes, for example
       // curl "example.com
       let prevEnd = node.type === "string" ? 1 : 2;
@@ -241,7 +241,7 @@ function toTokens(
       throw new CCError(
         "unexpected argument type " +
           JSON.stringify(node.type) +
-          '. Must be one of "word", "string", "raw_string", "ansii_c_string", "expansion", "simple_expansion", "string_expansion" or "concatenation"\n' +
+          '. Must be one of "word", "string", "raw_string", "ansi_c_string", "expansion", "simple_expansion", "translated_string" or "concatenation"\n' +
           underlineNode(node, curlCommand)
       );
   }
@@ -520,8 +520,8 @@ function extractCommandNodes(
   //       word |
   //       "string" |
   //       'raw_string' |
-  //       $'ansii_c_string' |
-  //       $"string_expansion" |
+  //       $'ansi_c_string' |
+  //       $"translated_string" |
   //       ${expansion} |
   //       $simple_expansion |
   //       concatenation)))

